### PR TITLE
Description arg -d, categories added, bug fixed encoding

### PR DESCRIPTION
**Pull request by janhoy**
Added category in brackets after program title
Possible to print detailed description for series and episodes with new param -d
Bug fixed encoding problem for SAVE_PATH and for printing episode titles
Improved help text. Now also prints usage if no args

### CODE REVIEW

File: nrkdl.py [hunk 1]
```diff
@@ -473,7 +475,15 @@ def _console(self, q):
         else:
             # use reverse since 0 is the closest match and i dont want to scoll
             for i, hit in reversed(list(enumerate(response['hits']))):
-                print('{0:>3}: {1}'.format(i, c_out(hit['hit']['title'])))
+                if 'category' in hit['hit']:
+                    category = c_out(hit['hit']['category']['title'])
+                else:
+                    category = ""
+                print('{0:>3}: {1} [{2}]'.format(i, c_out(hit['hit']['title']), category))
```
Comment (Hellowlol) [inline nrkdl.py]: Can you make sure that [] isnt printed if the categoey is missing. Just make it part of the categoey str or something

**Comment by Hellowlol** [comment]
Thanks for the pr. Did you run the tests manually and did it pass? I have been thinking about adding a description so this was just perfect.

**Comment by janhoy** [comment]
I'm not familiar with the testframework, so I did not run any automated tests at all. I may have messed up the tests with some of the changes...

**Comment by Hellowlol** [comment]
I use pytest as a test running. To run the test you just execute py.test in bash inside the nrkdl folder. Ps you didnt break any tests. I did some small changes to the pr to simplify the encoding (passing encodig shouldnt be needed. its only self as a option in case python gets it wrong, or the locale of the user is fucked up)

Thanks for the pr.

**Pull request merged**